Source/WebCore/platform/network/ResourceResponseBase.h

```diff
@@ -469,6 +497,129 @@ bool ResourceResponseBase::decode(Decoder& decoder, ResourceResponseBase& respon
     return true;
 }
 
+template<class Encoder, typename> void ResourceResponseBase::CrossThreadData::encode(Encoder& encoder) const
```

Comment:
This is used for storage, so maybe update the names accordingly.
Maybe mention that changing this should change the cache storage version number.

Reply:
I have moved this to WTF::Persistence, so it's more clear that it is used for storage and not supposed to be updated without checking disk storage.